src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -15,459 +16,72 @@
-import { VuetifyRuleFunction } from '@/types'
+import { SpecialResolutionIF } from '@bcrs-shared-components/interfaces/'
 import { useStore } from '@/store/store'
+import ResolutionEditor from './ResolutionEditor.vue'
+import SigningParty from './SigningParty.vue'
 
 @Component({
   components: {
-    HelpSection,
-    DatePickerShared
+    DatePickerShared,
+    HelpSpecialResolution,
+    InstructionalText,
+    ResolutionEditor,
+    SigningParty
   }
 })
-export default class CreateSpecialResolution extends Mixins(DateMixin) {
-  @Getter(useStore) getResource!: ResourceIF
-  @Getter(useStore) getBusinessFoundingDateTime!: string
-  @Getter(useStore) getCurrentDate!: string
-  @Getter(useStore) getSpecialResolution!: SpecialResolutionIF
+export default class CreateSpecialResolution extends Vue {
   @Getter(useStore) getComponentValidate!: boolean
+  @Getter(useStore) getSpecialResolution!: SpecialResolutionIF
   @Getter(useStore) getSpecialResolutionFormValid!: boolean
+  @Getter(useStore) showCreateSpecialResolution!: boolean
 
   @Action(useStore) setSpecialResolution!: ActionBindingIF
-  @Action(useStore) setValidComponent!: ActionBindingIF
 
-  // Refs
   $refs!: {
-    resolutionDatePickerRef: DatePickerShared,
-    signatureDatePickerRef: DatePickerShared,
-    createSpecialResolutionRef: FormIF
+    resolutionForm: HTMLFormElement
   }
 
-  // Date properties
-  protected resolutionDateText = ''
-  protected signingDate = ''
-
-  readonly MAX_RESOLUTION_TEXT_LENGTH = 1000
-  protected resolution = ''
-  protected formValid = false
-
-  protected signatory: PersonIF = {
-    givenName: '',
-    familyName: '',
-    additionalName: null
-  }
-
-  /** Validation rule for individual name fields */
-  readonly firstNameRules = this.nameRules('First Name')
-  readonly middleNameRules = this.nameRules('Middle Name', false)
-  readonly lastNameRules = this.nameRules('Last Name')
-
-  /** Validations rules for resolution date field. */
-  readonly resolutionDateRules = [
-    (v: string) => !!v || 'Resolution date is required'
-  ]
-
-  /** Validations rules for signing date field. */
-  readonly signatureDateRules = [
-    (v: string) => !!v || 'Signature date is required'
-  ]
-
-  get helpSection (): HelpSectionIF {
-    return this.getResource.changeData?.specialResolution?.helpSection || {}
-  }
-
-  get getSpecialResolutionResource (): SpecialResolutionSampleFormIF {
-    return this.getResource.changeData?.specialResolution?.sampleFormSection || {}
-  }
-
-  /** download URL for pdf file */
-  get documentURL (): string {
-    /**
-     * In session is stored the BASE_URL with business ID
-     * So we are taking from process.env.BASE_URL
-     */
-    return process.env.BASE_URL +
-      this.getSpecialResolutionResource?.path
-  }
-
-  /** The name section validity state (when prompted by app). */
+  /** The section validity state (when prompted by app). */
```

Comment:
Comment doesn't match code.

The other filings check `this.getComponentValidate` for whether to show the invalid section. This flag is enabled when the user clicks the button to go to the review page.

Reply:
(Corrections are different because they're a single page. In that case, validation should occur when the user clicks the File button.)

Reply:
sounds good

Reply:
Fixed, thanks.